doc/source/serve/api/rest_api.md

```diff
@@ -117,38 +124,194 @@ Content-Type: application/json
+                    "ray_actor_options": {"env_vars": {"override_increment": "5"}}
+                },
+            ]
+        }
+    ]
+}
 ```
 
+
+
 **Example Response**
 
+
 ```http
 HTTP/1.1 200 OK
 Content-Type: application/json
 ```
 
-(serve-rest-api-config-schema)=
+### `GET "/api/serve/applications/"`
+
+Gets cluster-level info and comprehensive details on all Serve applications deployed on the Ray cluster. See the [metadata schema](serve-rest-api-response-schema) for the response's JSON schema.
+
+```http
+GET /api/serve/applications/ HTTP/1.1
+Host: http://localhost:52365/
+Accept: application/json
+```
+
+**Example Response (not showing the full JSON)**:
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+    "proxy_location": "HeadOnly",
+    "http_options": {
+        "host": "127.0.0.1",
+        "port": 8000
+    },
+    "deploy_mode": "MULTI_APP",
+    "applications": {
+        "fruit_stand": {
+            "name": "fruit_stand",
+            "route_prefix": "/fruit",
+            "docs_path": null,
+            "status": "RUNNING",
+            "message": "",
+            "last_deployed_time_s": 1679952253.748111,
+            "deployed_app_config": "...",
+            "deployments": {
+                "fruit_app_MangoStand": {
+                    "name": "fruit_app_MangoStand",
+                    "status": "HEALTHY",
+                    "message": "",
+                    "deployment_config": "...",
+                    "replicas": [
+                        {
+                            "replica_id": "fruit_app_MangoStand#bSkrHK",
+                            "state": "RUNNING",
+                            "pid": 59350,
+                            "actor_name": "...",
+                            "actor_id": "...",
+                            "node_id": "...",
+                            "node_ip": "...",
+                            "start_time_s": 1679952254.3458009
+                        }
+                    ]
+                },
+            }
+        },
+    }
+}
+```
+
+### `DELETE "/api/serve/applications/"`
+
+Shuts down Serve and all applications running on the Ray cluster. Has no effect if Serve is not running on the Ray cluster.
+
+**Example Request**:
 
-## Config Schema
+```http
+DELETE /api/serve/applications/ HTTP/1.1
+Host: http://localhost:52365/
+Accept: application/json
+```
+
+**Example Response**
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+```
+
+(serve-rest-api-config-schema)=
+## Serve Config Schema
 
 ```{eval-rst}
-.. autopydantic_model:: ray.serve.schema.ServeApplicationSchema
+.. currentmodule:: ray.serve
+```
+
 
+```{eval-rst}
+.. autosummary::
+   :toctree: doc/
+
+   schema.ServeDeploySchema
+   schema.HTTPOptionsSchema
+   schema.ServeApplicationSchema
+   schema.DeploymentSchema
+   schema.RayActorOptionsSchema
 ```
 
-(serve-rest-api-status-schema)=
+(serve-rest-api-response-schema)=
+## Serve response schemas
```

Comment:
```suggestion
## Serve Response Schemas
```

Reply:
Although I'm a proponent of sentence case titles, the rest of the headers seem to be title case. Consistency is the priority.